config/config-api/src/main/java/com/thoughtworks/go/config/ConfigSaveValidationContext.java

```diff
@@ -283,6 +284,19 @@ public MaterialConfigs getAllMaterialsByFingerPrint(String fingerprint) {
         return matchingMaterials == null ? new MaterialConfigs() : matchingMaterials;
     }
 
+    @Override
+    public Map<CaseInsensitiveString, Boolean> getPipelinesWithMaterial(String fingerprint) {
```

Comment:
Don't you think method name is mis-leading?

Reply:
This is the implementation of the method in `ValidationContext`